scripts/ci/tools/freespace.py

```diff
@@ -0,0 +1,68 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""freespace.py for cean environment before to start CI"""
+
+from platform import version
+import shlex
+import subprocess
+import click
+from click import ClickException
+from rich.console import Console
+from typing import List
+
+console = Console(force_terminal=True, color_system="standard", width=180)
+
+option_verbose = click.option(
```

Comment:
I think what we need here, is https://click.palletsprojects.com/en/8.0.x/options/#values-from-environment-variables 

and we should get the `--verbose` option automatically use VERBOSE environment variable is specified. I think it would be great in CI to see "verbose" version of the "free_space" in CI. 

And since we already have:
```
VERBOSE: "true"
```
set as an environment variable in CI , we will avoid passing `--verbose` each time when we run the script in CI.

Reply:
Smth like:

```
click.option(
    "--verbose",
     envvar='VERBOSE',
...
```

Reply:
I got it: Now we can run: python3 freespace.py taking the VERBOSE variable environment. Thank you, Jarek.